cluster-autoscaler/processors/provreq/injector.go

```diff
@@ -29,26 +30,30 @@ import (
 	provreqconditions "k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/conditions"
 	provreqpods "k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/pods"
 	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/provreqclient"
+	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/provreqwrapper"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog/v2"
 	"k8s.io/utils/clock"
 )
 
 const (
-	defaultRetryTime = 10 * time.Minute
+	defaultRetryTime = 1 * time.Minute
+	maxBackoffTime   = 10 * time.Minute
 )
 
 // ProvisioningRequestPodsInjector creates in-memory pods from ProvisioningRequest and inject them to unscheduled pods list.
 type ProvisioningRequestPodsInjector struct {
-	client *provreqclient.ProvisioningRequestClient
-	clock  clock.PassiveClock
+	clock           clock.PassiveClock
+	client          *provreqclient.ProvisioningRequestClient
+	backoffDuration map[string]time.Duration
 }
 
 // Process pick one ProvisioningRequest, update Accepted condition and inject pods to unscheduled pods list.
 func (p *ProvisioningRequestPodsInjector) Process(
 	_ *context.AutoscalingContext,
 	unschedulablePods []*apiv1.Pod,
 ) ([]*apiv1.Pod, error) {
+	var retryTime time.Duration
```

Comment:
Super small nit: you probably don't need to declare `retryTime` here since it's only used within the following loop, and it's set to a new value each iteration. Feel free to disregard, just pointing out sth that would make it a bit more readable for me:)